src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtension.php

```diff
@@ -149,6 +150,23 @@ public function breadcrumbFunction($uuid)
     }
 
     /**
+     * @param string $requestUrl
+     * @param string $itemUrl
+     *
+     * @return bool
+     */
+    public function isActiveNavElementFunction($requestUrl, $itemUrl)
```

Comment:
call it `requestPath` and `itemPath`. because i think that would be the "perfect" to use the path for this check solution.